cms/models.py

```diff
@@ -145,6 +148,109 @@ def serve(self, request, *args, **kwargs):
         raise Http404
 
 
+class WebinarIndexPage(Page):
+    """
+    A placeholder page to group webinars under it as well as consequently add /webinars/ to the course page urls
+    """
+
+    slug = WEBINAR_INDEX_SLUG
+    template = "webinars_list_page.html"
+    parent_page_types = ["HomePage"]
+    subpage_types = ["WebinarPage"]
+
+    @classmethod
+    def can_create_at(cls, parent):
+        """
+        You can only create one of these pages under the home page.
+        The parent is limited via the `parent_page_type` list.
+        """
+        return (
+            super().can_create_at(parent)
+            and not parent.get_children().type(cls).exists()
+        )
+
+    def serve(self, request, *args, **kwargs):
+        """
+        We need to serve the webinars index template for list view.
+        """
+        return Page.serve(self, request, *args, **kwargs)
+
+    def get_context(self, request, *args, **kwargs):
+        """Populate the context with a dict of categories and live webinars"""
+        return dict(
+            **super().get_context(request),
+            **get_base_context(request),
+            webinars={
+                category: WebinarPage.objects.live().filter(category=category)
+                for category in [UPCOMING_WEBINAR, ON_DEMAND_WEBINAR]
+            },
+        )
+
+
+class WebinarPage(MetadataPageMixin, Page):
+    """
+    Webinar page model
+    """
+
+    parent_page_types = [WebinarIndexPage]
+
+    WEBINAR_CATEGORY_CHOICES = [
+        (UPCOMING_WEBINAR, UPCOMING_WEBINAR),
+        (ON_DEMAND_WEBINAR, ON_DEMAND_WEBINAR),
+    ]
+
+    category = models.CharField(max_length=20, choices=WEBINAR_CATEGORY_CHOICES)
+    banner_image = models.ForeignKey(
+        Image,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Image for the Webinar.",
+    )
+    start_datetime = models.DateTimeField(
+        null=True, blank=True, help_text="The start date and time of the webinar."
+    )
+    duration = models.PositiveIntegerField(
```

Comment:
Let's talk to @cachob to confirm if this really needs to be an integer field or this can be a text field like we have duration in course pages.

Reply:
This is how the time is displayed on the frontend.
`start_datetime` - `start_datetime + duration`, I have kept the field as minutes but minutes are not displayed on the frontend. This has an edge case that a 30-minute webinar will have a similar start/end time.

Reply:
Had a discussion with @arslanashraf7  and FYI @cachob:
[Designs](https://projects.invisionapp.com/d/main/#/console/22229357/471905085/inspect) show that we have a webinar start and end time. I have added 2 fields for that i.e. `start_datetime` and `duration`. 

- `start_datetime` is a date time field and is used to display the start time of the webinar like `Tuesday, May 2, 2023 | 11 AM`
- `duration` is an Integer Field that takes the duration of a webinar in `MINUTES`. This is used to calculate the end time of the webinar and display the end time.

I have a couple of questions:

- Do we want to keep the duration field an Integer field? If yes, what should we expect `MINUTES or HOURS?
- We can also separate out day and time of the webinar into 2 fields. The day would be a date field and the time will be a TEXT Field and you can add like `2 PM  - 3 PM ET` etc.

What are your thoughts on this @cachob?

Reply:
I think that would should keep the duration field as an integer and in minutes, with the logic that if the value is null or zero then it will not be displayed. Will this field be manually entered or is this dynamically generated based on the length of the recorded video? We should keep this manual, and leave it to the admin to determine if they want this shown or not for the pre-recorded / on-demand webinars.

As for the second question - let's separate day and time like you described so that the content admin can create this info as close to the design as possible

Reply:
Actually, duration is for the upcoming webinar and is used to determine the end time of a webinar using the start time. I think we can go with separate Fields for the day and time. The day will be a date field and the time will be a `TextField` where the admin can add the time and zone details.